packages/eas-json/schema/eas.schema.json

```diff
@@ -187,7 +187,12 @@
           "default": "default",
           "anyOf": [
             {
-              "enum": ["default", "medium", "large"]
+              "enum": ["default", "medium", "large"],
+              "markdownEnumDescriptions": [
+                "Maps to `\"default\"` for both Android and iOS",
```

Comment:
what does it mean?
Description for option default specifies that it maps to default?

Reply:
I meant that it maps to platform-specific `default` for both iOS and Android. Although it might be confusing for the reader. Do you have any suggestions about how to make it better?

Reply:
> meant that it maps to platform-specific default

If you are talking about our graphql values, then there is no such concept as platform-specific default/medium/large, stuff like e.g. android-medium are implementation details, and should not be acknowledged in anything user-facing.
If you are talking about specifying resource class in the root of a build profile vs under ios/android fields, then this is how the build profile works.

If you don't want to specify what default/medium/large actually is, and I assume you don't because we want to control those values server side, then I would just remove it. If you want to add description then maybe sth like
default - `default resource class`
medium - medium resource class
...